### ISSUE TITLE
[Raspberry] no audio latest RPI 3

### ISSUE THREAD

**Issue by steeley** [AlbrechtL/dab-rpi_raspbian_image#3]
Title says it all....

Seems to tune in stations better than an RPI 2 but still audio is dead until I reset alsa to force jack out like this

amixer cset numid=3 1
amixer cset numid=2 1

Sound plays but 128k stereo is all choppy. mono 80k just about plays.

**Comment by AlbrechtL** [comment]
Do you see the small rainbow rectangle in the right screen corner?
If yes then you Raspberry has a to low power source. Please use a strong 5 V source and a short and thick USB cable to power the RPI.
If you see the small rectangle dab-rpi will not work because the CPU throttles down and this can results in crappy audio.

**Comment by steeley** [comment]
No, power source is good. Htop shows heavy CPU (up to 100%) across all cores.
Will compare with SDR-J at some point.

**Comment by phil-audio** [comment]
hi , 
same problem no sound
and date always 01.01.2016 00:00 ?
https://nsa40.casimages.com/img/2019/03/09/19030905442661492.jpg

Rasp3  on Lan

**Comment by AlbrechtL** [comment]
Can you try it with the latest next version?
